server/app/boards.go

```diff
@@ -145,6 +147,38 @@ func (a *App) DuplicateBoard(boardID, userID, toTeam string, asTemplate bool) (*
 	if err != nil {
 		return nil, nil, err
 	}
+
+	// copy any file attachments from the duplicated blocks.
+	if err = a.CopyCardFiles(boardID, bab.Blocks); err != nil {
+		return nil, nil, fmt.Errorf("could not copy files while duplicating board %s: %w", boardID, err)
+	}
```

Comment:
In case of an error copying the files, should we delete the incomplete board and its blocks? Maybe something like:

```suggestion
	if err = a.CopyCardFiles(boardID, bab.Blocks); err != nil {
		if err := a.store.DeleteBoardsAndBlocks(bab, userID); err != nil {
			a.logger.Error("Cannot delete board after duplication error when copying block's files", mlog.String("boardID", bab.Boards[0].ID), mlog.Err(err)) 
		}
		return nil, nil, fmt.Errorf("could not copy files while duplicating board %s: %w", boardID, err)
	}
```